composer.json

```diff
@@ -16,13 +16,13 @@
 
     "require": {
         "php":                           ">=5.3.1",
-        "behat/mink":                    "~1.6@dev",
+        "behat/mink":                    "~1.7@dev",
```

Comment:
Well, this was to make it consistent with other drivers, but it is indeed unnecessary. We could even go further and remove this entirely, letting BrowserKitDriver bring Mink (this requirement was mainly useful to get the dev version of the testsuite)